mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.tsx

```diff
@@ -1,13 +1,22 @@
+import {useEffect, useState} from "react";
+import clsx from "clsx";
+import {User} from "firebase/auth";
 import {Accordion, accordionTypes} from "src/component/accordion/Accordion";
 import {AccordionContent} from "src/component/accordion/AccordionContent/AccordionContent";
 import {AccordionTrigger} from "src/component/accordion/AccordionTrigger/AccordionTrigger";
 import {HeadingLevel, Title} from "src/component/title/Title";
+import {handleUserAuthState} from "src/service/auth/handleUserAuthState";
 import styles from "src/logic/aboutProjectPage/AboutProjectPage.module.scss";
 
 /**
  * About project page
  */
 export const AboutProjectPage = () => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    handleUserAuthState(setUser);
+  }, []);
```

Comment:
Dont know clear but something looks wrong. It's strange that you using state for userAuth on each page. 
I think better delete title from all pages or move it to header